Src/java/tools/cql-formatter/build.gradle

```diff
@@ -3,5 +3,11 @@ apply plugin: 'application'
 mainClassName = 'org.cqframework.cql.tools.formatter.Main'
 
 dependencies {
-    compile project(':cql')
+    compile project(":cql-to-elm")
```

Comment:
Can we make sure this is only a test dependency? I want to make sure the cql formatter can be used with the cql-to-elm translator.